packages/eslint-plugin-udemy/CHANGELOG.md

```diff
@@ -3,7 +3,18 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-       <a name="6.0.0"></a>
+ <a name="6.0.1"></a>
+## [6.0.1](https://github.com/udemy/js-tooling/compare/eslint-plugin-udemy@6.0.0...eslint-plugin-udemy@6.0.1) (2018-09-12)
+
+
+### Bug Fixes
+
+* change location to decorator instead of method definition ([968d25e](https://github.com/udemy/js-tooling/commit/968d25e))
```

Comment:
Remember to start your Git commits with an upper-case letter, and the first line shouldn't end in a period. No biggie.